rate-limit-filter/src/lib.rs

```diff
@@ -16,14 +17,31 @@ pub fn _start() {
 }
 
 #[derive(Debug)]
-struct UpstreamCall {}
+struct UpstreamCall {
+    //paths: Vec<String>,
+}
 
 impl UpstreamCall {
     fn new() -> Self {
-        return Self {};
+        return Self {
+            //paths: retrieve().unwrap(),
+        };
     }
 }
 
+/*
+fn retrieve() -> Result<Vec<String>, Error> {
+    let mut client = Client::connect("host=localhost user=postgres dbname=mesherydb", NoTls)?;
+    let array = client
+        .query("SELECT PathName FROM Paths", &[])?
+        .iter()
+        .map(|x| x.get(0))
+        .collect();
+    Ok(array)
+}
+*/
+
+//to be removed
 static ALLOWED_PATHS: [&str; 4] = ["/auth", "/signup", "/upgrade", "/pull"];
```

Comment:
A to-do item on the list of things to generalize.

Reply:
Yup! This PR basically is meant to get the docs and image hub in a state where it's a piece of cake to get going. I've also added headers as @kumarabd requested a couple of days ago, so you should get good insights on the requests as well.